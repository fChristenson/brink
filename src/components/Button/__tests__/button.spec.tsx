import React from "react";
import { Button, styles } from "..";
import { childrenTests } from "../../../libs/tests/childrenTest";
import { baseTests } from "../../../libs/tests/baseTests";

describe("Button tests", () => {
  baseTests(<Button />, styles);
  childrenTests(<Button />);
});
