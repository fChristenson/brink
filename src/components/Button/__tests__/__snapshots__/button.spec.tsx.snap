// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests Base test should assert that the component matches the snapshot 1`] = `"<button style=\\"cursor:pointer;width:230px;height:50px;background:#00B5B9;border-radius:47px;color:white;font-family:Roboto;font-style:normal;font-weight:bolder;font-size:16px;line-height:16px;text-align:center;letter-spacing:0.75px\\"></button>"`;

exports[`Button tests Base test should assert that the component matches the snapshot 2`] = `
Object {
  "background": "#00B5B9",
  "borderRadius": "47px",
  "color": "white",
  "cursor": "pointer",
  "fontFamily": "Roboto",
  "fontSize": "16px",
  "fontStyle": "normal",
  "fontWeight": "bolder",
  "height": "50px",
  "letterSpacing": "0.75px",
  "lineHeight": "16px",
  "textAlign": "center",
  "width": "230px",
}
`;
